Sources/AccountSDKIOSWeb/Client/Client+AppTransfer.swift

```diff
@@ -5,6 +5,7 @@ extension Client {
     public enum AppTransfer {
         case preTransfer(_ clientId: String, _ accessGroup: String?)
         case postTransfer(accessGroup: String?, completion: (Result<Void, Error>) -> Void)
+        case postTransferFromOldSDK(accessGroup: String?)  /// Only needs to be invoked if the old SDK was used to store login data pre App Transfer.
```

Comment:
Logic works fine, but I think we should have consistency in methods. Now one throws and the second has a completion handler.